client/notifications/socket-handlers.ts

```diff
@@ -0,0 +1,29 @@
+import type { NydusClient, RouteInfo } from 'nydus-client'
+import { ReduxAction } from '../action-types'
+import { dispatch, Dispatchable } from '../dispatch-registry'
+import { addNotification } from './action-creators'
+
+const eventToAction: Record<string, (event: any) => Dispatchable<ReduxAction> | undefined> = {
```

Comment:
You should really declare interfaces for these things like I did elsewhere, it is *very hard* to type these things after they're `any`, you have to do a lot of code-reading and debugging and looking at websocket messages, it sucks. So just do it up-front so we (I) don't have to spend hours guessing at this stuff please :)